.eslintrc

```diff
@@ -23,6 +24,8 @@
     "jsx-quotes"           : [2, "prefer-single"],
     "max-len"              : [2, 120, 2],
     "object-curly-spacing" : [2, "always"],
-    "comma-dangle"         : "off"
+    "comma-dangle"         : "off",
```

Comment:
Maybe I'll leave it as it is at this moment so to follow the initial rules as much as I can. But I see it's convenient to use the comma after the last element, so in the next project I'll try to get used to it.